src/components/CreateShareStructure/ListShareClass.vue

```diff
@@ -1,14 +1,368 @@
 <template>
-<div></div>
+  <v-card flat id="share-structure">
+
+    <!-- Summary Section -->
+    <div id="share-summary" v-if="isSummary">
+      <!-- Summary Header -->
+      <div class="share-summary-header" >
+        <v-icon>mdi-file-tree</v-icon>
+        <label class="share-summary-header-title"><strong> Share Structure</strong></label>
+      </div>
+
+      <!-- Summary Warning -->
+      <div v-if="showErrorSummary" class="share-summary-invalid-message">
+        <span>
+          <v-icon color="#1976d2">mdi-information-outline</v-icon>
+          This step is not complete.
+          <router-link id="router-link" :to="{ path: '/create-share-structure', query: { showErrors: true } }">
+            Return to this step to complete it.
+          </router-link>
+        </span>
+      </div>
+    </div>
+
+    <v-data-table
+      :headers="headers"
+      :items="shareClasses"
+      :disable-pagination="true"
+      :disable-sort="true"
+      :hide-default-footer="true"
+    >
+      <template v-slot:item="row" class="share-data-table">
+
+        <!-- Share Class Rows-->
+        <tr :key="row.item.id" class="class-row" :class="{ 'class-row-has-series': row.item.series.length}">
+          <td class="list-item__title">{{row.item.name}}</td>
+          <td>{{row.item.maxNumberOfShares || 'No Maximum'}}</td>
+          <td>{{row.item.parValue ? `$${row.item.parValue}` : 'No Par Value'}}</td>
```

Comment:
This should use localized currency formatting... What if the selected currency isn't dollars?
Also on line 106.
Or, we could limit currencies to CAD and USD only... but need to check with Scott/BAs first.

Reply:
I already talked to BAs and Scott about this. Currency symbol wont be displayed in the table. The symbol has to be taken off

Reply:
They will be updating the docs and wireframes